src/act.obj.cpp

```diff
@@ -3595,25 +3610,28 @@ ACMD(do_holster)
 
   const char *madefor = "<error, report to staff>";
   
-  if (!holster_can_fit(cont, obj))
-    dontfit++;
+  if (GET_OBJ_TYPE(cont) == ITEM_CYBERWARE && GET_OBJ_VAL(cont, 0) == CYB_FINGERTIP)
+    madefor = "monowhips";
```

Comment:
I'm not sure what I was thinking here, it's been a while since I wrote this. Probably set it for the logging message that follows. with the intent to handle it there but forgot to remove it when I figured that I don't need to handle gating in `holster_can_fit()` at all.
The actual gating happens in `find_holster_that_fits_weapon()` where it will only return a fingertip if the weapon is a monowhip otherwise it goes for regular holsters (including if there's no figertip and weapon is monowhip). There was no need for redundant checking.